### DISCUSSION TITLE
Crazyflies become unstable and start crashing

### DISCUSSION THREAD

**Discussion by pthinesh** [USC-ACTLab/crazyswarm#393] [General]
Hello!

I have completed the installation and configuration of crazyswarm with Optitrack motion capture system.
Everytime when I run hello_world.py with one crazyflie. It couldn't properly takeoff and then flies to a random direction and crashed.

The following screen recording shows one of the test run,
https://youtu.be/nT_7sDPQ_eQ

Following are my configuration;
**hower_swarm.launch**
![Screenshot from 2021-06-16 20-20-17](https://user-images.githubusercontent.com/84022671/122212102-6fe2d700-cee2-11eb-9d9c-d68e3a032398.png)

**OptiTrack streaming configuration**
![Capture](https://user-images.githubusercontent.com/84022671/122212132-7709e500-cee2-11eb-8977-9b83a515bafc.PNG)

![PXL_20210616_112823771](https://user-images.githubusercontent.com/84022671/122212160-7e30f300-cee2-11eb-8974-96d0795ce997.jpg)

I would very much appreciate if anyone could help identify the cause of this issue.
Thanks

Note: 
1. When I fly the crazyflie individually using flowsequenceSync.py from crazyflie_ros, it works perfectly fine, so I guess that the drone does not have any defects.
2. I conducted the test with two different crazyflies, both crazyflies crashed after flying at a random direction.

**Comment by whoenig** [comment]
This very much looks like your coordinate system doesn't match the ROS coordinate system. To verify, visualize the position in rviz without flying (just by holding the Crazyflie in your hand and moving it around). Make sure that positive x is where the antenna is, positive z is when you move up, and positive y follows from the right-hand rule.

**Reply by pthinesh** [reply]
Thanks! Problem solved.
